### PR TITLE
tutorial on how to use application insights

### PR DESCRIPTION
**Pull request by monikavar**

### CODE REVIEW

File: docs/howto-view-telemetry-in-applicationinsights.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# How to view logs and metrics in Application Insights
```
Comment (marcschier) [inline docs/howto-view-telemetry-in-applicationinsights.md]: Should go into tutorials folder as tut-appinsights.md

Link to this from the readme.md in tutorials (second level, where the bullet points are)

File: docs/howto-use-applicationinsights-metrics.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+# How to use Application Insights metrics
```
Comment (marcschier) [inline docs/howto-use-applicationinsights-metrics.md]: This doc should probably go into a more broader "dev-guide.md", where we add more info on how to extend the platform

File: docs/howto-use-applicationinsights-metrics.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+To begin using it, please have a look at the following examples:
+
+### Counters
+
+Each invocation of `TrackEvent` would set a value of 1.
+
+```
+using Microsoft.Azure.IIoT.Diagnostics;
+
+public class EndpointSecurityAlerter : IEndpointRegistryListener, 							IApplicationRegistryListener
```
Comment (karok2m) [inline docs/howto-use-applicationinsights-metrics.md]: Delete tabs/spaces after `IEndpointRegistryListener`.

File: docs/howto-use-applicationinsights-metrics.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+```
+
+### Gauges
+
+Gauges can have any numeric value and change arbitrarily.
+
+```
+using Microsoft.Azure.IIoT.Diagnostics;
+
+public class EndpointSecurityAlerter : IEndpointRegistryListener, 							IApplicationRegistryListener
```
Comment (karok2m) [inline docs/howto-use-applicationinsights-metrics.md]: Delete tabs/spaces after `IEndpointRegistryListener`.

File: docs/howto-use-prometheus-metrics.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # How to use Prometheus metrics
```
Comment (karok2m) [inline docs/howto-use-prometheus-metrics.md]: I think this should also go to `dev-guides`.

File: docs/dev-guides/howto-use-applicationinsights-metrics.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
+# How to use Application Insights metrics
+
+[Home](readme.md)
+
+All the cloud microservices of Industrial IoT platform have been instrumented with Application Insights already.  The instrumentation monitors your app and sends telemetry data to Application Insights.
```
Comment (karok2m) [inline docs/dev-guides/howto-use-applicationinsights-metrics.md]: Please remove double space: `already.  The`

File: docs/tutorials/tut-applicationinsights.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+3. Select Time Range and click Run.
+
+   ![run](../media/appinsights3.png)
+   
+4. Application Insights queries are based on Kusto Query Language(KQL). Search for the specific log traces by writing your log queries. Learn more about writing queries [here](<https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview>).
+
+   ![query](../media/appinsights4.png)
+   
+5. It is possible to  filter on the results of the query on any column.
```
Comment (karok2m) [inline docs/tutorials/tut-applicationinsights.md]: Please remove double space: `to  filter`

File: docs/dev-guides/howto-use-applicationinsights-metrics.md (hunk 6 above)
Comment (monikavar) [reply docs/dev-guides/howto-use-applicationinsights-metrics.md]: I think so too.

File: docs/howto-use-prometheus-metrics.md (hunk 5 above)
Comment (monikavar) [reply docs/howto-use-prometheus-metrics.md]: I think so too.

**Pull request merged**